src/tequila/circuit/_gates_impl.py

```diff
@@ -255,8 +277,17 @@ def __pow__(self, power, modulo=None):
 
 class PowerGateImpl(ParametrizedGateImpl):
 
```

Comment:
Would be great to have a small doc string, that just comments on the power an parameter attributes